src/combinatorics/simple_graph/basic.lean

```diff
@@ -79,26 +74,20 @@ def simple_graph.from_rel {V : Type u} (r : V → V → Prop) : simple_graph V :
 noncomputable instance {V : Type u} [fintype V] : fintype (simple_graph V) :=
 by { classical, exact fintype.of_injective simple_graph.adj simple_graph.ext }
 
-@[simp]
-lemma simple_graph.from_rel_adj {V : Type u} (r : V → V → Prop) (v w : V) :
-  (simple_graph.from_rel r).adj v w ↔ v ≠ w ∧ (r v w ∨ r w v) :=
-iff.rfl
+@[simp] lemma simple_graph.from_rel_adj {V : Type u} (r : V → V → Prop) (v w : V) :
+  (simple_graph.from_rel r).adj v w ↔ v ≠ w ∧ (r v w ∨ r w v) := iff.rfl
 
-/--
-The complete graph on a type `V` is the simple graph with all pairs of distinct vertices adjacent.
--/
-def complete_graph (V : Type u) : simple_graph V :=
-{ adj := ne }
+/-- The complete graph on a type `V` is the simple graph with all pairs of distinct vertices
+adjacent -/
```

Comment:
I don't see the necessity of reflowing all these docstrings. Also, removing the periods from the ends of complete sentences seems like a step backwards.

Reply:
OK I have restored the periods at ends of sentences and followed the docstring convention

Reply:
Could you please split the cosmetic changes off into a separate PR?  I'm sorry to ask you to do this extra work, but it makes it easier to review new features.  I think you'd use the `chore(simple_graph)` heading for this.  (But also, it tends to be better to leave style alone unless there is something in the styleguide saying otherwise.  There is nothing specifying that attributes should be on the same line or that documentation comments should be flowed like this.)

Reply:
Thanks for doing this!